<h2>layout: basic
title: &quot;Résumé&quot;
date: 2018-11-28T15:15:34+10:00
permalink: &quot;/resume/&quot;
description: &quot;My Curriculum Vitae&quot;</h2>
<p><a href="/assets/files/Giovanni_Visona_CV.pdf" target="_blank" 
    rel="noopener" class=" button-primary">
Download CV in pdf format
<i class="fa fa-download ml-1"></i>
</a></p>
<h1>Current Position</h1>
<h3>Biomedical AI/ML Engineer</h3>
<h5>2024 - Ongoing</h5>
<h5>Heidelberg, Germany</h5>
<h4>GSK.ai</h4>
<blockquote>
<p>For confidentiality reasons, I cannot provide overly specific descriptions of the application fields on which I focus.</p>
</blockquote>
<ul>
<li>Research and development of state-of-the-art ML models to optimize performance on a range of biomedical prediction tasks, leading to models that are being considered for registration as medical devices.</li>
<li>Consistently placed top 3 in my department for both code contributions and code reviews in the past 4 quarters.</li>
<li>Implemented bespoke support tools to enhance job scheduling and experiment analysis for my team. This resulted in an improved throughput of training jobs by a factor of ∼ 3.</li>
<li>Liaised with experts in biology, medicine, and clinical trials to ensure optimal collection of data to train biomedical ML models.</li>
<li>Secure handling of sensitive data.</li>
</ul>
<hr>
<h1>Skills, Technologies, and Scientific Expertise</h1>
<p><strong>Programming Languages:</strong> Python, R, Go, SQL</p>
<p><strong>Tools:</strong> Pytorch, Pandas, Polars, Ibis, DuckDB, FastAPI, Scikit-learn, SQLite, HDF5, Git, Github, Docker, Kubernetes (CKAD-certified), Spark, Airflow</p>
<p><strong>Software Engineering:</strong> CI/CD (Azure, Github Actions), Cloud Computing (GCP)</p>
<p><strong>Machine learning and Data Science:</strong> Deep Learning, Reinforcement Learning, Classical ML (GLMs, Trees, GAMs, etc.), Diffusion Models, EDA, Data Visualization, Interpretable ML, Graph ML, Data Modelling</p>
<p><strong>Probability and Statistics:</strong> Hypothesis testing, A/B testing, Linear Algebra</p>
<p><strong>Biology and Medicine:</strong> Epigenetics, Genomics, Proteins, Pathways, Molecular Networks, Immunology, Small Molecules, Molecular Dynamics, Clinical Data (EHRs), Antimicrobial Resistance, Mass Spectrometry</p>
<hr>
<h1>Experience</h1>
<h3>ESR Researcher in Machine Learning for Precision Medicine</h3>
<h5>2019 - 2024</h5>
<h5>Tubingen, Germany</h5>
<h4>Max Planck Institute for Intelligent Systems</h4>
<ul>
<li>Designed and implemented deep-learning-based models and probabilistic models to solve problems in biology and biomedicine.</li>
<li>Published as first author or shared first author in internationally renowned journals, including Nature Communications, Bioinformatics, and Briefings in Bioinformatics.</li>
<li>ESR in the Marie Curie Innovative Training Network entitled “Machine Learning Frontiers in Precision Medicine”</li>
</ul>
<hr>
<h3>Junior Developer and Consultant</h3>
<h5>2016 - 2018</h5>
<h5>Padova, Italy</h5>
<h4>Espedia Consulting</h4>
<ul>
<li>Contributed to the creation of customized software solutions for clients, prioritizing robustness in design and ensuring on-time delivery.</li>
</ul>
<hr>
<h1>Education</h1>
<h3>PhD in Computer Science</h3>
<h5>2019 - 2025</h5>
<h5>Tübingen, Germany</h5>
<h4>University of Tübingen</h4>
<ul>
<li>Thesis: &quot;Biomedical Machine Learning Beyond the Training Distribution&quot;. Supervisors: Prof. Bernhard Schölkopf and Dr. Gabriele Schweikert.</li>
</ul>
<h3>MSc in Artificial Intelligence</h3>
<h5>2018 - 2019</h5>
<h5>Edinburgh, Scotland</h5>
<h4>University of Edinburgh</h4>
<ul>
<li>Graduated with Distinction.</li>
<li>Thesis: &quot;Optimising Recommendation Slates Using Deep Determinantal Point Processes&quot;<br>
Supervisors: Dr. Roberto Pellegrini and Aleksandr Petrov.</li>
</ul>
<h3>Master's Degree in Physics</h3>
<h5>2014 - 2016</h5>
<h5>Trento, Italy</h5>
<h4>University of Trento</h4>
<ul>
<li>Graduated with 110/110 marks with honours.</li>
<li>Thesis: &quot;Polymer Templating of Porous Silicon for Drug Delivery Applications&quot;<br>
Supervisor: Dr. Paolo Bettotti.</li>
</ul>
<h3>Bachelor's Degree in Physics</h3>
<h5>2012 - 2014</h5>
<h5>Torino, Italy</h5>
<h4>Università di Torino</h4>
<ul>
<li>Graduated with 110/110 marks with honours.</li>
<li>Thesis: &quot;Modelization of Nano Amplified Targeted Therapy (nATT)&quot;<br>
Supervisor: Prof. Cristiana Peroni<br>
Collaborator: Dr. Andrea Attili.</li>
</ul>
<h1>Publications</h1>
<p><strong>2026</strong>
<strong>Generalizable machine learning models for rapid antimicrobial resistance prediction in unseen healthcare settings</strong>
<em>GigaScience</em>, DOI: <a href="https://doi.org/10.1093/gigascience/giaf156">10.1093/gigascience/giaf156</a></p>
<hr>
<p><strong>2025</strong>
<strong>Towards personalized epigenomics: learning shared chromatin landscapes and joint de-noising of histone modification assays</strong>
<em>NAR Genomics and Bioinformatics</em>, DOI: <a href="https://doi.org/10.1093/nargab/lqaf188">10.1093/nargab/lqaf188</a></p>
<hr>
<p><strong>2024</strong><br>
<strong>Network propagation for GWAS analysis: a practical guide to leveraging molecular networks for disease gene discovery</strong><br>
<em>Briefings in Bioinformatics</em>, DOI: <a href="https://doi.org/10.1093/bib/bbae014">10.1093/bib/bbae014</a></p>
<hr>
<p><strong>2023</strong><br>
<strong>Multimodal learning in clinical proteomics: enhancing antimicrobial resistance prediction models with chemical information</strong><br>
<em>Bioinformatics</em>, DOI: <a href="https://doi.org/10.1093/bioinformatics/btad717">10.1093/bioinformatics/btad717</a></p>
<hr>
<p><strong>2023</strong>
<strong>A historical perspective of biomedical explainable AI research</strong><br>
<em>Patterns</em>, DOI: <a href="https://doi.org/10.1016/j.patter.2023.100830">10.1016/j.patter.2023.100830</a></p>
<hr>
<p><strong>2023</strong><br>
<strong>Getting personal with epigenetics: towards individual-specific epigenomic imputation with machine learning</strong><br>
<em>Nature Communications</em>, DOI: <a href="https://doi.org/10.1038/s41467-023-40211-2">10.1038/s41467-023-40211-2</a></p>
<hr>
<p><strong>2023</strong><br>
<strong>Machine-Learning-Aided Prediction of Brain Metastases Development in Non–Small-Cell Lung Cancers</strong><br>
<em>Clinical Lung Cancer</em>, DOI: <a href="https://doi.org/10.1016/j.cllc.2023.08.002">10.1016/j.cllc.2023.08.002</a></p>
<hr>
<p><strong>2017</strong><br>
<strong>Targeted dose enhancement in radiotherapy for breast cancer using gold nanoparticles, part 2: a treatment planning study</strong><br>
<em>Medical Physics</em>, DOI: <a href="https://doi.org/10.1002/mp.12178">10.1002/mp.12178</a></p>
<hr>
<p><strong>2017</strong><br>
<strong>Targeted dose enhancement in radiotherapy for breast cancer using gold nanoparticles, part 1: A radiobiological model study</strong><br>
<em>Medical Physics</em>, DOI: <a href="https://doi.org/10.1002/mp.12180">10.1002/mp.12180</a></p>
